detox/detox.d.ts

```diff
@@ -1740,9 +1740,11 @@ declare global {
             value?: unknown;
         }
 
-        interface IosElementAttributeFrame {
-            y: number;
+        interface IosElementAttributeFrame extends ElementAttributeFrame { }
+
+        interface ElementAttributeFrame {
             x: number;
+            y: number;
```

Comment:
This is okay but can be better. I think we need to either get rid of `IosElementAttributeFrame` (and use `ElementAttributeFrame`, strictly) or introduce `AndroidElementAttributeFrame`. The former might be a breaking change, though. @noomorph wdyt?

Reply:
Indeed, we don't break typings to adhere to the semantic versioning policy. Do your best not to change the signatures dramatically. It's better to accumulate a list of breaking changes and do it in one focused leap.

Reply:
Leaving it like that